Gems/Multiplayer/Code/Source/NetworkEntity/EntityReplication/EntityReplicator.cpp

```diff
@@ -172,6 +172,7 @@ namespace Multiplayer
             m_onEntityStopHandler.Disconnect();
             netBindComponent->AddEntityStopEventHandler(m_onEntityStopHandler);
             AttachRPCHandlers();
+            netBindComponent->NetworkActivated();
```

Comment:
nit: It seems strange that the entity replicator "owns" telling an entity that the network is activated. Are there other places this event could live or is there a clear reason why this is owned by EntityReplicators?

Reply:
The specific thing we care about is AttachRPCHandlers right above it. That function is what actually makes RPC dispatch functional for a networked entity.